### ISSUE TITLE
Windows: ensure $HOME/bin exists and is in $PATH

### ISSUE THREAD

**Issue by jokeyrhyme** [jokeyrhyme/dotfiles-rs#28]
We need somewhere generic to put individual executable files

See:

-   https://msdn.microsoft.com/en-us/library/windows/desktop/ms682653(v=vs.85).aspx

-   https://stackoverflow.com/questions/2121795/programmatically-modifiy-environment-variables

**Comment by jokeyrhyme** [comment]
https://www.windowscentral.com/how-edit-registry-using-command-prompt-windows-10

**Comment by jokeyrhyme** [comment]
```
C:\Windows\system32>reg query HKEY_CURRENT_USER\Environment /v Path

HKEY_CURRENT_USER\Environment
    Path    REG_EXPAND_SZ    C:\Users\Ron\.cargo\bin;...
```

**Comment by jokeyrhyme** [comment]
```
C:\Windows\system32>reg add HKEY_CURRENT_USER\Environment /v Path /t REG_EXPAND_SZ /d "C:\Users\Ron\.cargo\bin;...;C:\Users\Ron\bin"
Value Path exists, overwrite(Yes/No)? y
The operation completed successfully.
```

**Comment by jokeyrhyme** [comment]
Fixed by #30 
For now, we don't try to modify the Windows Registry ourselves, we just flag that it needs manual modification and `panic!()`
